sysbuild/CMakeLists.txt

```diff
@@ -93,22 +241,33 @@ function(${SYSBUILD_CURRENT_MODULE_NAME}_post_cmake)
   cmake_parse_arguments(POST_CMAKE "" "" "IMAGES" ${ARGN})
   restore_ncs_vars()
 
+  set(SYSBUILD y)
```

Comment:
The scripts need to know that they are running in sysbuild mode, this can be considered a `toup` that needs to move to zephyr at some point

Reply:
Correction: this will not be getting submitted upstream. The reason it is needed is this code is called as a function without access to the `SYSBUILD` variables, some of the cmake code has had different code flow paths added when sysbuild is used and this is needed so that those alternative paths run